src/index.js

```diff
@@ -1,7 +1,114 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import L from 'leaflet';
+ReactDOM.render(	
+  <h1>Hello World!</h1>,	
+ document.getElementById('root')
+) 
 
-ReactDOM.render(
-  <h1>Hello World!</h1>,
-  document.getElementById('root')
-)
\ No newline at end of file
+var fs = require('fs');
```

Comment:
Everything from lines 9 through 50 can be moved to a new file. Remember that your Express server must be contained in a separate file to avoid compatibility complications with React and to improve the readability of your code.